src/index.ts

```diff
@@ -8,6 +8,7 @@ export * from './compact';
 export * from './concat';
 export * from './countBy';
 export * from './createPipe';
+export * from './deepMege';
```

Comment:
There's a typo here: `deepMege` vs `deepMerge`.

Also, to allow importModularization (e.g. in [nextJs](https://nextjs.org/docs/architecture/nextjs-compiler#modularize-imports)), it is necessary that each re-export is done from a separate file with the exact same name, so `deepMergeLeft` and `deepMergeRight` would need to be split apart please.

Reply:
Fixed typo and split into two files